man/fi_gni.7.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: page
+title: fi_gni(7)
+tagline: Libfabric Programmer's Manual
+---
+{% include JB/setup %}
+
+# NAME
+
+The GNI Fabric Provider
+
+# OVERVIEW
+
+The GNI provider runs on Cray XC Systems utilizing the user-space
+Generic Network Interface (uGNI) which provides low-level access to
+the Cray XC's Aries Interconnect.  The Aries interconnect is designed
+for low-latency one-sided messaging and also includes direct hardware
+support for common atomics operations and optimized collectives.
+
+# REQUIREMENTS
+
+The GNI provider run on Cray XC Systems running CLE 5.2 UP02 or
+higher using gcc version 4.9 or higher.
+
+# SUPPORTED FEATURES
+
+The GNI provider supports the following features defined for the
+libfabric API:
+
+*Endpoint types*
+: The provider supports the *FI_EP_MSG* endpoint type.
```

Comment:
I'm going to blame that type-o on having written that part at 6:30am on the plane.